src/domWrapper.ts

```diff
@@ -142,9 +110,11 @@ export class DOMWrapper<ElementType extends Element> {
       if (tagName === 'SELECT') {
         return this.trigger('change')
       }
-      this.trigger('input')
+      await this.trigger('input')
       // trigger `change` for `v-model.lazy`
-      return this.trigger('change')
+      await this.trigger('change')
+
+      return
```

Comment:
This is an API change

Reply:
This is a breaking change - you *must* await `setValue` now.

Although this change is probably good, I am not sure we want to introduce a breaking change. Let's see what others thinks.

Our docs cover when and why to use `await` extensively, this would require updating those a lot, too.

Reply:
reverted

Reply:
👍 Let's keep this reverted for now (so the PR does not introduce more changes than expected), but maybe making `setValue` async all the time is… a good thing? 🤔